templates/1-basic_workflow_with_config/config1.yml

```diff
@@ -1,29 +1,43 @@
-# example of a config for the workflow
+# Example of a config file for the workflow
 
+# The contents of this file control the workflow. The highest level entries are the tasks to accomplish.
+# Below the task level are the elements required for the task. For example, the task 'extract' interates
+# over 'reconstruction' data and 'simulation' data, the two sections specifying what (data, variables, time periods) 
+# to extract and how (region, attributes, subsequent saving) to do that. For the 'regrid' tasks, among other, 
+# the 'input' and 'output' data are specified along with the methodology for the regridding process.
 
-# This workflow is made to know exactly where to restart if the code is stopped.
-# It looks if the result of a task exists in the project catalog before executing the task.
-# The workflow does NOT automatically remove intermediate files. You might run out of space.
```

Comment:
why remove this ?

Reply:
It's not removed but rephrased, see line 13ff.

Reply:
oups missed it! thanks!